app/src/main/java/com/trunghoang/orderhub/ui/mainScreen/MainScreenFragment.kt

```diff
@@ -0,0 +1,76 @@
+package com.trunghoang.orderhub.ui.mainScreen
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageButton
+import androidx.appcompat.widget.PopupMenu
+import androidx.appcompat.widget.Toolbar
+import androidx.fragment.app.Fragment
+import com.trunghoang.orderhub.R
+import kotlinx.android.synthetic.main.fragment_main_screen.view.*
+import kotlinx.android.synthetic.main.toolbar_main_screen.view.*
+
+class MainScreenFragment() : Fragment(), PopupMenu.OnMenuItemClickListener {
+    companion object {
+        @JvmStatic
+        fun newInstance(
+            supportToolbarCallback: SupportToolbarCallback,
+            logoutCallback: LogoutCallback
+        ) = MainScreenFragment().apply {
+            this.mSupportToolbarCallback = supportToolbarCallback
+            this.mLogoutCallback = logoutCallback
+        }
+    }
+
+    lateinit var mContext: Context
+    var mSupportToolbarCallback: SupportToolbarCallback? = null
+    var mLogoutCallback: LogoutCallback? = null
+
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        mContext = context
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_main_screen, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        mSupportToolbarCallback?.setToolbar(view.toolbarMainScreen)
+        view.navigationMainScreen
+            ?.getHeaderView(0)
+            ?.findViewById<ImageButton>(R.id.buttonSettings)
```

Comment:
a thu dung buttonSettings.setOnclickListenter(this ) xem có được k. cái này e cungx k chắc :)

Reply:
không được đâu vì header là 1 phần tử trong recyclerview mặc định của navigationView, nên ko lấy button ra trực tiếp bằng cách đó được